src/App.jsx

```diff
@@ -70,13 +79,19 @@ function App() {
       <ThemeProvider theme={theme}>
         <CssBaseline />
         <Router>
-          <Header />
-          <Switch>
-            {routes.map(({ path, component, title }) => (
-              <RouteWithTitle key={path} {...{ path, component, title }} />
-            ))}
-          </Switch>
-          <Footer />
+          <LastLocationProvider>
+            <Header />
+            <Switch>
+              {routes.map(({ path, component, title, guard }) => (
+                <RouteWithTitle
+                  key={path}
+                  {...{ path, component, title, guard }}
+                />
+              ))}
+            </Switch>
+            <Route path="/" component={LoginDialog} />
```

Comment:
What does this do? We already have a route for this path on line 66.

Reply:
This is so that the login dialog appears, I couldn't get it to show without it making it a route.